docs/platforms/javascript/common/sourcemaps/uploading/angular-nx.mdx

```diff
@@ -70,6 +65,23 @@ We recommend you add the auth token to your CI/CD environment as an environment
 SENTRY_AUTH_TOKEN=___ORG_AUTH_TOKEN___
 ```
 
+### Webpack
+
+Install the Sentry Webpack plugin:
```

Comment:
```suggestion
If you're using Webpack and haven't already, install the Sentry Webpack plugin:
```
Just felt odd to assume they don't have it installed yet

Reply:
I think it's fair to assume they haven't yet installed the plugin. This page is a sourcemaps uploading guide like the usual ones.